src/auxkernels/DensityNormalization.C

```diff
@@ -17,6 +17,8 @@ DensityNormalization::validParams()
 {
   InputParameters params = AuxKernel::validParams();
   params.addRequiredCoupledVar("Density", "The variable to be normalized in log form");
+  params.deprecateParam("Density", "density", "04/01/2026");
```

Comment:
```suggestion
```

As discussed this morning, don't worry about a deprecation timeline for this one. Let's just do it.